ui/app/routes/vault/cluster/access/mfa/index.js

```diff
@@ -0,0 +1,37 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+export default class MfaRoute extends Route {
+  @service router;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+    },
+  };
+
+  model(params) {
+    return this.store
+      .lazyPaginatedQuery('mfa-method', {
+        responsePath: 'data.keys',
+        page: params.page || 1,
+      })
+      .then((model) => {
+        return model;
+      })
```

Comment:
The promise will resolve the response so unless you need to do something with it the `.then` is not needed.